connectors/netty-connector/src/test/java/org/glassfish/jersey/netty/connector/FollowRedirectsTest.java

```diff
@@ -37,12 +38,20 @@
 import org.glassfish.jersey.netty.connector.internal.RedirectException;
 import org.glassfish.jersey.server.ResourceConfig;
 import org.glassfish.jersey.test.JerseyTest;
+import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 
 public class FollowRedirectsTest extends JerseyTest {
 
     private static final Logger LOGGER = Logger.getLogger(FollowRedirectsTest.class.getName());
     private static final String TEST_URL = "http://localhost:9998/test";
+    private static final AtomicReference<String> TEST_URL_REF = new AtomicReference<>();
+
+    @BeforeEach
+    public void before() {
+        final String url = TEST_URL.replace("9998", String.valueOf(getPort()));
```

Comment:
why not use pattern formatting instead of replacing? 
```
private static final String TEST_URL_PATTERN = "http://localhost:%d/test";
...
String.format(TEST_URL_PATTERN, getPort());
```